src/elements/content-sidebar/SidebarNavButton.js

```diff
@@ -38,13 +38,15 @@ const SidebarNavButton = ({
                 const sidebarState = { open: isToggle ? !isOpen : true };
 
                 return (
-                    <Tooltip position="middle-left" text={tooltip}>
+                    <Tooltip position="middle-left" text={tooltip} isTabbable={false}>
                         <NavButton
                             aria-selected={isActive()}
+                            aria-controls={`${dataTestId}-content`}
                             activeClassName="bcs-is-selected"
                             className="bcs-NavButton"
                             data-resin-target={dataResinTarget}
                             data-testid={dataTestId}
+                            id={dataTestId}
```

Comment:
Feels a bit weird that we are mixing something that is used for testing with things needed for accessibility.